package.json

```diff
@@ -6,7 +6,7 @@
   "module": "es/index.js",
   "jsnext:main": "es/index.js",
   "scripts": {
-    "test": "node_modules/.bin/jest",
+    "test": "cross-env NODE_ENV=commonjs node_modules/.bin/jest",
```

Comment:
(Ignore me -- I was working off of a clean repo by accident.)

Reply:
Does this mean we're not testing our `es` build, though?

Reply:
There's no way to directly test the ES build in node, since it uses `import`. If we translate the `import`s back to `require`s, then we just end up with the original compiled code, which does get tested.